collector-backend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG base_image_tag
+FROM concordium/base:${base_image_tag} as build
+EXPOSE 8080
+WORKDIR /build
+COPY . .
+ARG build_profile
+RUN cargo build --"${build_profile}"
+
+FROM ubuntu:20.04
+ARG build_profile
+COPY --from=build /build/target/"${build_profile}"/node-collector-backend /node-collector-backend
```

Comment:
Isn't the name of the binary just `collector-backend`? That's what's in the version I see.

Reply:
No this is the name I see.